packages/rum/src/domain/segmentCollection/deflateWorker.js

```diff
@@ -4442,6 +4443,70 @@ function workerCodeFn() {
       return deflate$1(input, options)
     }
 
-    return { Deflate, constants }
+    // https://github.com/nodeca/pako/blob/master/lib/utils/strings.js#L26
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏since master is a moving reference, it could be interesting to reference the exact commit from we used.